device/globals/player.gd

```diff
@@ -100,7 +100,12 @@ func anim_get_ph_paths(p_anim):
 	return ret
 
 func play_anim(p_anim, p_notify = null, p_reverse = false, p_flip = null):
-	if p_notify != null && (!has_node("animation") || !get_node("animation").has_animation(p_anim)):
+	if not animation:
+		vm.report_errors("player", ["Animation not found for play_anim"])
+	if not animation.has_animation(p_anim):
+		vm.report_errors("player", [animation.name + " does not contain " + p_anim])
+
+	if p_notify != null && (!animation || !animation.has_animation(p_anim)):
```

Comment:
Before someone points out the redundancy here, I opted to keep the `!animation` checks in place, because not everyone necessarily has configured their projects to crash on `vm.report_errors`.